packages/pcb-stackup/example/index.js

```diff
@@ -2,7 +2,7 @@
 
 const fs = require('fs')
 const path = require('path')
-const {promisify} = require('util')
+const promisify = require('pify')
```

Comment:
FYI: Node v4 has reached end-of-life. https://github.com/nodejs/Release#end-of-life-releases

Reply:
Oh damn definitely misread the year in that chart. Thanks for this, will drop v4 from test matrix